<?xml version="1.0" encoding="UTF-8" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom" xmlns:dc="http://purl.org/dc/elements/1.1/">
  <channel>
    <title>Error after updading to 1.0.9</title>
    <link>http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658</link>
    <description>Hi,

I have just updated to the latest Release and now I have this error, which I absolutely don&#39;t know what to do with :confused:
all my prevously saved patches don&#39;t work now. Can someone tell me what to do?

&lt;img src=&quot;/uploads/default/original/2X/1/1922400c02d6a37786314fc147c5dcbc5a96b842.jpg&quot; width=&quot;690&quot; height=&quot;259&quot;&gt;</description>
    
    <lastBuildDate>Mon, 09 May 2016 16:10:39 +0000</lastBuildDate>
    <category>Helpdesk</category>
    <atom:link href="http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658.rss" rel="self" type="application/rss+xml" />
      <item>
        <title>Error after updading to 1.0.9</title>
        <dc:creator><![CDATA[@AdmiralCrunch]]></dc:creator>
        <description><![CDATA[
          <p><a href="http://community.axoloti.com/u/admiralcrunch">@AdmiralCrunch</a> wrote:</p>
          <blockquote>
              <p>tanks <img src="/images/emoji/emoji_one/slight_smile.png?v=2" title=":slight_smile:" class="emoji" alt=":slight_smile:"> works now :))</p>
          </blockquote>
          <p><a href="http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658/4">Read full topic</a></p>
        ]]></description>
        <link>http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658/4</link>
        <pubDate>Mon, 09 May 2016 16:10:39 +0000</pubDate>
        <guid isPermaLink="false">community.axoloti.com-post-1658-4</guid>
        <source url="http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658.rss">Error after updading to 1.0.9</source>
      </item>
      <item>
        <title>Error after updading to 1.0.9</title>
        <dc:creator><![CDATA[@thetechnobear Mark Harris]]></dc:creator>
        <description><![CDATA[
          <p><a href="http://community.axoloti.com/u/thetechnobear">@thetechnobear</a> wrote:</p>
          <blockquote>
              <p>fixed, and sync libs should resolve...</p>

<p>(though i fear a conflict if you have started modifying the file system, if you do get this you will need to reinit)</p>

<p>this is caused by a change in code in 1.0.10, which means its fine in 1.0.10, not in 1.09<br>.... pertinent to the discussions we are having in developer/object lifecycle..</p>
          </blockquote>
          <p><a href="http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658/3">Read full topic</a></p>
        ]]></description>
        <link>http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658/3</link>
        <pubDate>Mon, 09 May 2016 11:53:03 +0000</pubDate>
        <guid isPermaLink="false">community.axoloti.com-post-1658-3</guid>
        <source url="http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658.rss">Error after updading to 1.0.9</source>
      </item>
      <item>
        <title>Error after updading to 1.0.9</title>
        <dc:creator><![CDATA[@DrJustice]]></dc:creator>
        <description><![CDATA[
          <p><a href="http://community.axoloti.com/u/drjustice">@DrJustice</a> wrote:</p>
          <blockquote>
              <p>That error also occurs here after a library sync. To get rid of it, delete the file:</p>

<p>C:\Users\Documents\axoloti\axoloti-contrib\objects\tb\env\cycle ad.axo</p>

<p>I'm sure that object will be fixed in no time flat, once tb spots this thread <img src="/images/emoji/emoji_one/slight_smile.png?v=2" title=":slight_smile:" class="emoji" alt=":slight_smile:"></p>
          </blockquote>
          <p><a href="http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658/2">Read full topic</a></p>
        ]]></description>
        <link>http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658/2</link>
        <pubDate>Sun, 08 May 2016 20:21:51 +0000</pubDate>
        <guid isPermaLink="false">community.axoloti.com-post-1658-2</guid>
        <source url="http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658.rss">Error after updading to 1.0.9</source>
      </item>
      <item>
        <title>Error after updading to 1.0.9</title>
        <dc:creator><![CDATA[@AdmiralCrunch]]></dc:creator>
        <description><![CDATA[
          <p><a href="http://community.axoloti.com/u/admiralcrunch">@AdmiralCrunch</a> wrote:</p>
          <blockquote>
              <p>Hi,</p>

<p>I have just updated to the latest Release and now I have this error, which I absolutely don't know what to do with <img src="/images/emoji/emoji_one/confused.png?v=2" title=":confused:" class="emoji" alt=":confused:"><br>all my prevously saved patches don't work now. Can someone tell me what to do?</p>

<p><div class="lightbox-wrapper"><a data-download-href="//community.axoloti.com/uploads/default/1922400c02d6a37786314fc147c5dcbc5a96b842" href="//community.axoloti.com/uploads/default/original/2X/1/1922400c02d6a37786314fc147c5dcbc5a96b842.jpg" class="lightbox" title="Unbenannt-1.jpg"><img src="//community.axoloti.com/uploads/default/optimized/2X/1/1922400c02d6a37786314fc147c5dcbc5a96b842_1_690x259.jpg" width="690" height="259"><div class="meta">
<span class="filename">Unbenannt-1.jpg</span><span class="informations">1657x624 625 KB</span><span class="expand"></span>
</div></a></div></p>
          </blockquote>
          <p><a href="http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658/1">Read full topic</a></p>
        ]]></description>
        <link>http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658/1</link>
        <pubDate>Sun, 08 May 2016 19:31:17 +0000</pubDate>
        <guid isPermaLink="false">community.axoloti.com-post-1658-1</guid>
        <source url="http://community.axoloti.com/t/error-after-updading-to-1-0-9/1658.rss">Error after updading to 1.0.9</source>
      </item>
  </channel>
</rss>
